### ISSUE TITLE
Tree System Suggestion

### ISSUE THREAD

**Issue by Retrogamebloke** [TheGameCreators/GameGuruRepo#1561]
I have been playing with the new tree tools, the're quite good but perhaps an extra option to avoid putting them in water areas might be an idea for a future addition. I say optional, because it may be useful for some scenes of course like a swamp for instance; but generally you won't want trees under water and it can be quite difficult to see them to remove them manually.
![image](https://user-images.githubusercontent.com/44620846/141813806-4cffd547-2044-4f1a-b0c8-1986c60fee90.png)

**Comment by rvanner** [comment]
@Retrogamebloke The system should stop trees populating under the water line. Did you raise the water after you placed the trees down?

**Comment by Retrogamebloke** [comment]
No. I raised it before adding anything else to get an island type look. I only tried the trees afterwards as speed on my compo entry was falling through the floor. I can send the map if it will help.

**Comment by rvanner** [comment]
I think the trees are populates based on the terrain blend positions. Seems to make sense to also take into consideration the height of the water.
